deploy/devdeploy.py

```diff
@@ -23,12 +24,23 @@ def _setup_database(cur, database: str, schema: str, stage: str):
 
 def _copy_dependencies(cur, schema: str, stage: str):
     print(f"Copying dependencies to @{schema}.{stage}.")
-    for file in ["sqlglot.zip"]:
+    # Writes all dependencies into `@stage/python`
+    for file in ["sqlglot.zip", "crud.zip"]:
         local_file_path = f"app/python/{file}"
         stage_file_path = f"@{schema}.{stage}/python"
         put_cmd = f"PUT 'file://{local_file_path}' '{stage_file_path}' overwrite=true auto_compress=false"
+        print(put_cmd)
         cur.execute(put_cmd)
 
+    # Also copy the CRUD zip into @stage/ui because the streamlit app cannot reach "out" of the `ui` directory
+    # to dynamically load this from elsewhere in the stage.
+    put_cmd = f"PUT 'file://app/python/crud.zip' '@{schema}.{stage}/ui' overwrite=true auto_compress=false"
+    print(put_cmd)
+    cur.execute(put_cmd)
+
+    # Copy the CRUD zip from app/python to app/ui to keep devdeploy and deploy streamlit consistent with each other.
+    shutil.copy2('app/python/crud.zip', 'app/ui/crud.zip')
```

Comment:
@rymurr curious what you think here. We have a couple of ways we could make the crud module available to the streamlit app for local dev.

* Set `PYTHONPATH` when doing `run streamlit app/ui/Home.py`
* Use the symlink `ln -s ../crud app/ui/crud`
* Let devdeploy.py rebuild the zip and use the zip (only) in the app

The last one is what this PR is doing because it keeps us from having to do "are we running locally or inside a native app" check to conditionally load the `crud.zip` module. Any strong opinions for v1?